tools/quicktype_runner.js

```diff
@@ -12,11 +12,31 @@ const {
   JSONSchema,
   JSONSchemaInput,
   JSONSchemaStore,
-  TypeScriptTargetLanguage,
   parseJSON,
   quicktype,
+  splitIntoWords,
```

Comment:
This is obsolete when https://github.com/quicktype/quicktype/pull/1119#event-1987297419 is released